atspi-common/src/operation.rs

```diff
@@ -10,25 +10,27 @@ pub enum Operation {
 	Insert,
 	#[serde(rename = "delete")]
 	#[serde(alias = "delete/system")]
+	#[serde(alias = "remove")]
+	#[serde(alias = "remove/system")]
 	Delete,
 }
 
 impl TryFrom<&str> for Operation {
 	type Error = crate::AtspiError;
 	fn try_from(s: &str) -> Result<Operation, Self::Error> {
 		match s {
-			"add" | "add/system" => Ok(Operation::Insert),
-			"delete" | "delete/system" => Ok(Operation::Delete),
-			_ => Err(crate::AtspiError::KindMatch(format!("\"{s}\" is not a type of Operation"))),
+			"add" | "add/system" | "insert" | "insert/system" => Ok(Operation::Insert),
+			"delete" | "delete/system" | "remove" | "remove/system" => Ok(Operation::Delete),
+			_ => Err(crate::AtspiError::KindMatch(format!("{s} is not a type of Operation"))),
 		}
 	}
 }
 
 impl From<Operation> for String {
 	fn from(op: Operation) -> String {
```

Comment:
I left a comment on this implementation on an earlier incarnation of this, which now seems hidden.

As rule of thumb, generally prefer implementing `Display` over `From<Operation> for String` because `Display` is more versatile: [It allows for writing in any buffer - even in no `alloc` situations.  Also, `.to_string` comes for free with `Display` implementations.](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2021&gist=8153b380f51064a65ef5b43bbcac1f50)

Demonstrated it on the playground.